bin/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+if [ -f "${DOCKER_CERT_PATH}/ca.pem" ]
```

Comment:
I think if you find that you really need to wrap ContainerPilot itself, then ContainerPilot has a bug or missing feature.

If you find that you really need to wrap the application rather than using a `preStart`, then ContainerPilot probably has a bug or missing feature (although realistically all the cases I've seen of this so far will be covered by the improved dependency handling in CPv3).